### ISSUE TITLE
Error in Vector3.snapped()

### ISSUE THREAD

**Issue by cbscribe** [godotengine/godot#18050]
**Godot version:**
3.0.2

**Issue description:**
API Documentation for Vector3 states

> Vector3 snapped ( Vector3 by )

but using a Vector3 argument results in 

> Invalid type in function 'snapped' in base 'Vector3'. Cannot convert argument 1 from Vector3 to float.

So it seems to be expecting a float argument. However, passing any float argument results in no error but also no change to the source vector.

I would expect it to work like `Vector2.snapped()`.

**Steps to reproduce:**

    var v = Vector3(1.212, 1.923, 1.009).snapped(Vector3(0.01, 0.01, 0.01)))

**Comment by Noshyaar** [comment]
Can't reproduce 9e7cee2
```
func _ready():
	var v = Vector3(1.212, 1.923, 1.009).snapped(Vector3(0.01, 0.01, 0.01))
	print(v)
```

prints `(1.21, 1.92, 1.01)`

**Comment by cbscribe** [comment]
Yes, I just built from master and confirmed working as expected. Must have been fixed since 3.0.2 release.

**Comment by akien-mga** [comment]
It was fixed in the master branch by #16362. Since it breaks compat we decided not to cherry-pick it in 3.0.x, but it would be worth mentioning in the 3.0 documentation that the method is broken..
